README.md

```diff
@@ -18,10 +18,198 @@ Or install it yourself as:
 
     $ gem install litmus_paper
 
+## Overview
+
+Litmus Paper reports health for each service on a node on a 0-100 scale. Health is computed by aggregating the values returned from running various subchecks.
+
+There are two classes of subchecks: Dependencies and Metrics. Dependencies report either 0 or 100 health, and aggregate such that if any dependency is not 100 the whole service is down. Metrics report health on a scale from 0-100 and aggregate as averages based on their weight.
+
+You can also force a service to report a health value on a host.
+Forcing a service up makes it report a health of 100, regardless of the measured health.
+Forcing a service down makes it report a health of 0.
+Forcing a service's health to a value between 0 and 100 causes it to report that value.
```

Comment:
I think it's worth explaining that forcing a health sets a maximum health, but if the reported health is lower it will still return that.  @ahayworth explains the logic well in ec1b34ab3ba27e99e0e069304167192384e25ccb

Reply:
That's a great point and I'm glad you remembered it. Because I didn't. :)